lib/unit/utils/http_helper.rb

```diff
@@ -6,8 +6,9 @@
 
 module HttpHelper
   VALUES = [:"filter[searchRadius]"].freeze
-  def self.get(url, headers:, params: nil)
-    make_request(Net::HTTP::Get, url, headers, params: params)
+  RESPONSE_TYPES = %w[delete image].freeze
```

Comment:
can you explain where you got this response type from?

Reply:
I created it for differentiating types of response as when we delete smth or get image as response, standard Json parsing can't work. Then we need to receive unparsed response, that is why I added additional checks.